test/kubemark/start-kubemark.sh

```diff
@@ -45,41 +45,18 @@ KUBECTL="${KUBE_ROOT}/cluster/kubectl.sh"
 KUBEMARK_DIRECTORY="${KUBE_ROOT}/test/kubemark"
 export RESOURCE_DIRECTORY="${KUBEMARK_DIRECTORY}/resources"
 export SHARED_CA_DIRECTORY=${SHARED_CA_DIRECTORY:-"/tmp/shared_ca"}
```

Comment:
suggest to keep the kube-mark changes in different PR, to keep this PR just for kube-up to support scale-out which should not include any kubemark changes.

Reply:
Kubemark also call kube-up to deploy cluster. If remove kubemark change in this PR, kubemark will be totally blocked.